client/src/game/rules/segment.ts

```diff
@@ -22,6 +22,7 @@ export function shouldStore(
     action.type === "gameOver" &&
     action.endCondition !== EndCondition.Timeout &&
     action.endCondition !== EndCondition.Terminated &&
+    action.endCondition !== EndCondition.VotedToKill &&
```

Comment:
this has to be added at the end or else all the idle timeouts in the database will show as the wrong thing